apps/website/src/pages/hackathon/[id].astro

```diff
@@ -2,346 +2,310 @@
+					<h2>Submit an app to earn rewards</h2>
+
+					<Grid columns={3}>
+						{hackathon.rewardsData.map((reward) => (
+							<Card title={reward.title || ''}>
+								{reward.image?.public_id && (
+									<img
+										slot="img"
+										src={`https://res.cloudinary.com/jlengstorf/image/upload/w_400,f_auto,q_auto/${reward.image.public_id}`}
+										alt={reward.title || 'Reward'}
+									/>
+								)}
+								<p>{reward.description}</p>
+							</Card>
+						))}
+					</Grid>
+				</Block>
+			)
+		}
 
-			<List>
-				{
-					data.rules.map((rule) => (
-						<Card title={rule.title}>
-							<p>{rule.description}</p>
-						</Card>
-					))
-				}
+		{
+			primarySponsor && (
+				<Block name="sponsor" emphasized>
+					<div class="inline">
+						<p>
+							This challenge is brought to you by{' '}
+							<strong>{primarySponsor.title}</strong>
+						</p>
+
+						{primarySponsor.link && (
+							<a href={primarySponsor.link} class="button secondary">
+								learn more &rarr;
+							</a>
+						)}
+					</div>
+				</Block>
+			)
+		}
 
-				<div class="list-fine-print">
-					{
-						data.rules
-							.filter((r) => r.finePrint)
-							.map((rule) => <p>{rule.finePrint}</p>)
-					}
-				</div>
-			</List>
-		</Block>
+		{
+			hackathon.resources && hackathon.resources.length > 0 && (
+				<Block name="resources" variant="two-column-asymmetrical">
+					<div class="heading">
+						<h2>Resources</h2>
+					</div>
+
+					<List>
+						{hackathon.resources.map((resource) => (
+							<Card variant="secondary">
+								<p slot="title">
+									<a href={resource.url || '#'}>{resource.title} &rarr;</a>
+								</p>
+								<p>{resource.description}</p>
+							</Card>
+						))}
+					</List>
+				</Block>
+			)
+		}
 
-		<Block name="submit" emphasized>
-			<h2>Show us what you got!</h2>
-		</Block>
+		{
+			hackathon.rules && hackathon.rules.length > 0 && (
+				<Block name="rules" variant="two-column-asymmetrical">
+					<div class="heading">
+						<h2>The Rules</h2>
+					</div>
+
+					<List>
+						{hackathon.rules.map((rule, index) => (
+							<Card title={`#${index + 1}: ${rule.title}`}>
+								<p>{rule.description}</p>
+							</Card>
+						))}
+					</List>
+				</Block>
+			)
+		}
 
-		<Block name="faq" variant="two-column-asymmetrical">
-			<div class="heading">
-				<h2>FAQ</h2>
-			</div>
+		{
+			hackathon.submissionForm && (
+				<Block name="submit" emphasized>
+					<div class="heading">
+						<h2>Show us what you built!</h2>
+					</div>
+					<div class="content">
+						<div class="text-container">
+							<p>
+								When your app is ready, click this button to submit your app.
+								Apps must be submitted through this form before the deadline to
+								qualify for rewards.
+							</p>
+						</div>
+					</div>
+					<div class="button-container">
+						<a href={hackathon.submissionForm} class="button">
+							Submit Your App
+						</a>
+					</div>
+				</Block>
+			)
+		}
```

Comment:
for now, we need logic to hide this when the hackathon is closed. in the future we need something else. maybe a showcase of apps or a CTA to see active hackathons